int16 gain
int16 exposure